.gitignore

```diff
@@ -53,3 +53,9 @@ src/mixxx.rc.include
 src/mixxx.res
 src/test/**/*.actual
 res/qrc_mixxx.cc
+.vs/CMake Overview
```

Comment:
Please refer to the preamble in this file that explains why those additions are neither needed nor desired.